### PR TITLE
feat: visual adjustments to the page preview panel

### PR DESCRIPTION
**Pull request by neatbyte-vnobis**
## Changes
Implements issue "Visual adjustments to the page preview panel"

SHOULD BE MERGED AFTER https://github.com/webiny/webiny-js/pull/3103

## How Has This Been Tested?
Manual

## Documentation
None

### CODE REVIEW

**Comment by SvenAlHamad** [comment]
I've tested this one and noticed a small css issue, the dropdown arrow goes outside the dropdown container: 
CleanShot 2023-03-16 at 19 52 10@2x

**Comment by neatbyte-vnobis** [comment]
@SvenAlHamad I have checked this, and I cannot reproduce the issue.

Maybe you need to rebuild UI on this branch? (there was similar issue in December when you saw a similar bug locally, but after rebuild it was gone). Also please let me know env OS and used browser, if issue is still there.

**Comment by adrians5j** [comment]
Looks good on my end: 
image

image

File: packages/app-page-builder/src/admin/plugins/pageDetails/header/Header.tsx [hunk 1]
```diff
@@ -2,46 +2,100 @@ import React from "react";
...
         <React.Fragment>
             <HeaderTitle>
-                <PageTitle>
-                    <Typography use="headline5">{page.title}</Typography>
-                </PageTitle>
+                <PageInfo>
+                    <PageTitle>
+                        <Typography use="headline6">{page.title}</Typography>
+                    </PageTitle>
+                    {isSiteRunning && (
```
Comment (adrians5j) [inline packages/app-page-builder/src/admin/plugins/pageDetails/header/Header.tsx]: Can we rather have the link always shown, but when clicking on it, if the site is not running, display that modal that we show in other places? Like when we click on the View page in dropdown:
image

**Review by adrians5j — CHANGES_REQUESTED** [review]
Left a question in the code.

Also, two extra questions.

First one for @SvenAlHamad , should we have a different color for unpublished pages?

At the moment, we use green for published, everything else is yellow.

image

Second question - what happens with statuses when APW is enabled? I mean, will this work with APW enabled? Not sure how statuses work there. Can we expect statuses other than the default ones that come with basic version of Webiny?

**Comment by SvenAlHamad** [comment]
Let's use a darker grey color for "unpublished" pages. 
As for the APW question @adrians5j - APW still works with those same statuses, it's not affecting them.

**Review by adrians5j — APPROVED** [review]

**Pull request merged**